tests/test-lib-github.js

```diff
@@ -0,0 +1,27 @@
+const chai = require('chai');
```

Comment:
Cool le test :+1: 

Éventuellement on pourrait en rajouter un plus _intégration_ dans `tests/test-onboarding.js` qui serait presque un copié-collé de celui en ligne 189 "should not call Github API if the github username field is an url", genre :

```javascript
it('should not call Github API if the github username starts with "@"' (done) => {
      chai.request(app)
        .post('/onboarding')
        .type('form')
        .send({
          firstName: 'Férnàndáô',
          lastName: 'Úñíbe',
          role: 'Dev',
          start: '2020-01-01',
          end: '2021-01-01',
          status: 'Independant',
          github: '@ghUsername',
        })
        .end((err, res) => {
          this.getGithubMasterSha.called.should.be.false;
          this.createGithubBranch.called.should.be.false;
          this.createGithubFile.called.should.be.false;
          this.makeGithubPullRequest.called.should.be.false;
          done();
        });
    });
```

Reply:
Enfin... quoique... déjà le test existant couvre ce cas car on fait le check de l'http dans la même fonction... peut être overkill du coup

Reply:
j'aime beaucoup l'idée, mais je pense que c'est overkill en effet :grin: 

dans le sens de la pyramide des tests, comme c'est déjà couvert par un test unitaire, et que pas mal de fonctionnalités sont testées par les tests d'intégrations (on parlait de ce concept dans le channel d'équipe, c'est pour ça que je me permets d'en reparler ici :slightly_smiling_face: ) 

![image](https://user-images.githubusercontent.com/4059615/105200223-e64ff880-5b3f-11eb-8666-7a80b9bd631e.png)

Reply:
Tout à fait d'accord